homeassistant/components/recorder/migration.py

```diff
@@ -64,6 +64,7 @@ def _apply_update(engine, new_version):
         # Create indexes for states
         _create_index(engine, "states", "ix_states_last_updated")
         _create_index(engine, "states", "ix_states_entity_id_created")
+        _create_index(engine, "states", "ix_states_created_domain")
```

Comment:
@balloob thanks! Looking at it, is the correct way to acheive this:
- Update models.py#SCHEMA_VERSION to 3
- Change migration.py#_applyUpdates to include a new block
`elif new_version == 3:` with my index migration

Anything else?